Detectors/FIT/FT0/workflow/include/FT0Workflow/FT0DataReaderDPLSpec.h

```diff
@@ -23,7 +23,7 @@
 #include "Framework/WorkflowSpec.h"
 #include "Framework/SerializationMethods.h"
 #include "DPLUtils/DPLRawParser.h"
-
+#include "Framework/InputRecordWalker.h"
```

Comment:
Is it needed? I don't see it used in this file

Reply:
Here https://github.com/AliceO2Group/AliceO2/blob/5b2f22a95c4a8cb791448174400fdd559b9150bd/Detectors/FIT/FT0/workflow/include/FT0Workflow/FT0DataReaderDPLSpec.h#L51

Reply:
BTW, this commit was partially copied from PR #5786